### ISSUE TITLE
Validations print errors on Arch Linux

### ISSUE THREAD

**Issue by invidian** [mrvladus/Errands#152]
Maybe you find it useful:

```console
==> Making package: errands 45.1.5-1 (Thu 11 Jan 2024 09:58:56 CET)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Updating Errands git repo...
==> Validating source files with sha256sums...
    Errands ... Skipped
==> Removing existing $srcdir/ directory...
==> Extracting sources...
  -> Creating working copy of Errands git repo...
Cloning into 'Errands'...
done.
Switched to a new branch 'makepkg'
==> Sources are ready.
==> Making package: errands 45.1.5-1 (Thu 11 Jan 2024 09:59:05 CET)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Removing existing $pkgdir/ directory...
==> Starting build()...
+ exec meson setup --prefix /usr --libexecdir lib --sbindir bin --buildtype plain --auto-features enabled --wrap-mode nodownload -D b_lto=true -D b_pie=true -D python.bytecompile=1 Errands build
The Meson build system
Version: 1.3.1
Source dir: /home/invidian/.cache/yay/errands/src/Errands
Build dir: /home/invidian/.cache/yay/errands/src/build
Build type: native build
Project name: errands
Project version: 45.1.5
Host machine cpu family: x86_64
Host machine cpu: x86_64
Found pkg-config: YES (/usr/bin/pkg-config) 2.1.0
Run-time dependency libadwaita-1 found: YES 1.4.2
Run-time dependency pygobject-3.0 found: YES 3.46.0
Program python3 found: YES (/usr/bin/python3)
Program python3 found: YES (/usr/bin/python3)
Configuring io.github.mrvladus.List.desktop.in using configuration
Program msgfmt found: YES (/usr/bin/msgfmt)
Program desktop-file-validate found: YES (/usr/bin/desktop-file-validate)
Configuring io.github.mrvladus.List.metainfo.xml.in using configuration
Program appstreamcli found: YES (/usr/bin/appstreamcli)
Configuring io.github.mrvladus.List.gschema.xml using configuration
Program glib-compile-schemas found: YES (/usr/bin/glib-compile-schemas)
Build-time dependency gio-2.0 found: YES 2.78.3
Program /usr/bin/glib-compile-resources found: YES (/usr/bin/glib-compile-resources)
Configuring errands using configuration
Program msginit found: YES (/usr/bin/msginit)
Program msgmerge found: YES (/usr/bin/msgmerge)
Program xgettext found: YES (/usr/bin/xgettext)
Dependency gio-2.0 found: YES 2.78.3 (cached)
Program /usr/bin/glib-compile-schemas found: YES (/usr/bin/glib-compile-schemas)
Program gtk4-update-icon-cache found: YES (/usr/bin/gtk4-update-icon-cache)
Program update-desktop-database found: YES (/usr/bin/update-desktop-database)
Build targets in project: 25

errands 45.1.5

  User defined options
    auto_features     : enabled
    buildtype         : plain
    libexecdir        : lib
    prefix            : /usr
    sbindir           : bin
    wrap_mode         : nodownload
    python.bytecompile: 1
    b_lto             : true
    b_pie             : true

Found ninja-1.11.1 at /usr/bin/ninja
INFO: autodetecting backend as ninja
INFO: calculating backend command to run: /usr/bin/ninja -C /home/invidian/.cache/yay/errands/src/build
ninja: Entering directory `/home/invidian/.cache/yay/errands/src/build'
[22/22] Merging translations for data/io.github.mrvladus.List.metainfo.xml
==> Starting check()...
ninja: Entering directory `/home/invidian/.cache/yay/errands/src/build'
ninja: no work to do.
1/3 Validate desktop file        OK              0.08s
2/3 Validate schema file         OK              0.02s
3/3 validate-appdata             FAIL            0.20s   exit status 3
>>> MALLOC_PERTURB_=51 ASAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1 UBSAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1:print_stacktrace=1 /usr/bin/appstreamcli validate --no-net --explain io.github.mrvladus.List.metainfo.xml
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― ✀  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
I: io.github.mrvladus.List:8: developer-name-tag-deprecated
   The toplevel `developer_name` element is deprecated. Please use the `name` element in a
   `developer` block instead.

I: io.github.mrvladus.List:65: description-first-para-too-short
     Todo application for those who prefer simplicity.
   The first `description/p` paragraph of this component might be too short (< 80 characters).
   Please consider starting with a longer paragraph to improve how the description looks like in
   software centers and to provide more detailed information on this component immediately in the
   first paragraph.

E: io.github.mrvladus.List:231: description-para-markup-invalid b
   This description paragraph contains invalid markup. Currently, only <em/> and <code/> are
   permitted.

✘ Validation failed: errors: 1, infos: 2, pedantic: 3
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――


Summary of Failures:

3/3 validate-appdata      FAIL            0.20s   exit status 3

Ok:                 2
Expected Fail:      0
Fail:               1
Unexpected Pass:    0
Skipped:            0
Timeout:            0


```

**Comment by mrvladus** [comment]
Thanks, I fixed errors in bbf1d8ad1b3acad341e16aebe07ed57db11a4879

**Comment by invidian** [comment]
Cool. I guess it can be closed then :)
